doc/keepalive.md

```diff
@@ -4,25 +4,27 @@ The keepalive ping is a way to check if a channel is currently working by sendin
 
 This guide documents the knobs within gRPC core to control the current behavior of the keepalive ping.
 
-The keepalive ping is controlled by two important channel arguments -
+The send-side keepalive ping in core is controlled by the following channel arguments -
```

Comment:
I believe this is labeled "client-side" in the IMPORTANT NOTE below. Should we choose one or the other term for consistency? Or is there a further distinction between them?

Reply:
"client-side" actually differs from "send-side" here. For example, these args GRPC_ARG_KEEPALIVE_TIME_MS and others are also valid on the server side, while the "server-side" args are args that only affect servers.

Reply:
I misunderstood, thanks for clarifying. Send-side sounds fine.

Reply:
I actually removed "send-side" since it is confusing that way